CONTRIBUTING.md

```diff
@@ -24,8 +24,6 @@ Before making a non-trivial change, please discuss it via [issues]. You should b
 1. Make changes to the codebase
 1. Before pushing, fix any errors possibly emitted by the following commands:
 
-   - `yarn format` fixes stylistic issues using [Prettier]
-   - `yarn lint` enforces coding rules based on the [Airbnb JavaScript Style Guide]
```

Comment:
These should be left as-is. Better do more than required instead of getting errors only while commiting. Please revert all the changes of this file.